src/components/MenuBar.vue

```diff
@@ -225,6 +294,10 @@ export default {
 			return this.lastImagePath
 				|| this.filePath.split('/').slice(0, -1).join('/')
 		},
+		token() {
```

Comment:
Might be good to call it sharingToken, as there is a separate token for the session within the text codebase as well, to avoid any confusion